webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -314,225 +318,9 @@ private static PdfReader createPdf(String globalTitle, Date creationDate, Scramb
-                cb.endText();
-
-
-                cb.beginText();
-                fontSize = 11;
-                cb.setFontAndSize(bf, fontSize);
-                cb.showTextAligned(PdfContentByte.ALIGN_CENTER, "DO NOT FILL IF YOU ARE THE COMPETITOR", competitorInfoLeft + (right-competitorInfoLeft)/2, top-offsetTop, 0);
-                offsetTop += fontSize + marginBottom;
-                cb.endText();
-
-                cb.beginText();
-                fontSize = 11;
-                cb.setFontAndSize(bf, fontSize);
-                cb.showTextAligned(PdfContentByte.ALIGN_CENTER, "Graded by: _______________ Result: ______", competitorInfoLeft + (right-competitorInfoLeft)/2, top-offsetTop, 0);
-                offsetTop += fontSize + marginBottom;
-                cb.endText();
-
-                cb.beginText();
-                cb.setFontAndSize(bf, 25f);
-                int MAGIC_NUMBER = 40; // kill me now
-                cb.showTextAligned(PdfContentByte.ALIGN_CENTER, "Fewest Moves", left+(competitorInfoLeft-left)/2, top-MAGIC_NUMBER, 0);
-                cb.endText();
-
-                com.itextpdf.text.List rules = new com.itextpdf.text.List(com.itextpdf.text.List.UNORDERED);
-                rules.add("Notate your solution by writing one move per bar.");
-                rules.add("To delete moves, clearly erase/blacken them.");
-                rules.add("Face moves F, B, R, L, U, and D are clockwise.");
-                rules.add("Rotations x, y, and z follow R, U, and F.");
-                rules.add("' inverts a move; 2 doubles a move. (e.g.: U', U2)");
-                rules.add("w makes a face move into two layers. (e.g.: Uw)");
-                rules.add("A [lowercase] move is a cube rotation. (e.g.: [u])");
-
-                ct.addElement(rules);
-                int rulesTop = competitorInfoBottom+55;
-                ct.setSimpleColumn(left+padding, scrambleBorderTop, competitorInfoLeft-padding, rulesTop, 0, Element.ALIGN_LEFT);
-                ct.go();
-
-                rules = new com.itextpdf.text.List(com.itextpdf.text.List.UNORDERED);
-                rules.add("You have 1 hour to find a solution.");
-                rules.add("Your solution length will be counted in OBTM.");
-                int maxMoves = WCA_MAX_MOVES_FMC;
-                rules.add("Your solution must be at most " + maxMoves + " moves, including rotations.");
-                rules.add("Your solution must not be directly derived from any part of the scrambling algorithm.");
-                ct.addElement(rules);
-                MAGIC_NUMBER = 150; // kill me now
-                ct.setSimpleColumn(left+padding, scrambleBorderTop, rulesRight-padding, rulesTop-MAGIC_NUMBER, 0, Element.ALIGN_LEFT);
-                ct.go();
-
-                doc.newPage();
+                addFMCSolutionSheet(docWriter, doc, scrambleRequest, globalTitle, i);
+                addFMCScrambleSheet(docWriter, doc, scrambleRequest, globalTitle, i);
```

Comment:
1. Do we want to include the generic fmc solution sheet and the separate fmc scramble sheets in the big pdf with every scramble?
2. If no to 1), then we need to generate a separate pdf file for a generic fmc solution sheet and separate pdf files for fmc scramble sheets (plural!).

I believe the answer to 1) should be "no". I hope that this will address @Laura-O's concerns [here](https://github.com/cubing/tnoodle/pull/224#issuecomment-244532072).

From reading this code, it looks like the fmc scramble sheets will end up in our big pdf. Could you instead move this out to `createPdf()` so we only generate separate files for them?